packages/fast-components/src/slider/slider.ts

```diff
@@ -0,0 +1,327 @@
+import { attr, FastElement, observable, Observable } from "@microsoft/fast-element";
+import { FormAssociated } from "../form-associated";
+import {
+    keyCodeArrowRight,
+    keyCodeArrowLeft,
+    keyCodeArrowDown,
+    keyCodeArrowUp,
+} from "@microsoft/fast-web-utilities";
+import { convertPixelToPercent } from "./slider-utilities";
+
+export enum SliderMode {
+    singleValue = "single-value",
+    adjustFromLower = "adjust-from-lower",
+    adjustFromUpper = "adjust-from-upper",
+    dualThumb = "dual-thumb",
```

Comment:
If we aren't supporting all of this out of the box it may be best to not include it until it's supported. Slider mode is simply "singleValue" which is the default.

Reply:
ok that makes sense

Reply:
done